Packs/AWS-Athena/Integrations/AWS-Athena/AWS-Athena.yml

```diff
@@ -1,56 +1,83 @@
 commonfields:
-  id: AWS - Athena - Beta
+  id: AWS - Athena
   version: -1
-name: AWS - Athena - Beta
-display: AWS - Athena (Beta)
+name: AWS - Athena
+display: AWS - Athena
 category: IT Services
-description: Amazon Web Services Athena
+description: Amazon Web Services Athena.
 configuration:
 - display: Role Arn
   name: roleArn
-  defaultvalue: ""
   type: 0
   required: false
+  section: Connect
 - display: Role Session Name
   name: roleSessionName
-  defaultvalue: ""
-  type: 0
-  required: false
-- display: AWS Default Region
-  name: defaultRegion
-  defaultvalue: ""
   type: 0
   required: false
+  section: Connect
 - display: Role Session Duration
   name: sessionDuration
-  defaultvalue: ""
   type: 0
   required: false
+  section: Connect
+  advanced: true
+- display: AWS Default Region
+  name: defaultRegion
+  type: 15
+  required: false
+  options:
+  - us-east-1
+  - us-east-2
+  - us-west-1
+  - us-west-2
+  - ca-central-1
+  - eu-west-1
+  - eu-central-1
+  - eu-west-2
+  - ap-northeast-1
+  - ap-northeast-2
+  - ap-southeast-1
+  - ap-southeast-2
+  - ap-south-1
+  - sa-east-1
+  - eu-north-1
+  - eu-west-3
+  - us-gov-east-1
+  - us-gov-west-1
+  section: Connect
 - display: Access Key
   name: credentials
   type: 9
   displaypassword: Secret Key
-  required: false
-- display: Access Key
-  name: access_key
-  defaultvalue: ""
+  required: true
+  section: Connect
+- display: Timeout
+  name: timeout
+  defaultvalue: "60,10"
   type: 0
-  hidden: true
   required: false
-- display: Secret Key
-  name: secret_key
-  type: 4
-  defaultvalue: ""
-  hidden: true
+  additionalinfo: The time in seconds until a timeout exception is reached. You can specify just the read timeout (for example 60) or also the connect timeout followed after a comma (for example 60,10). If a connect timeout is not specified, a default of 10 second will be used.
+  section: Connect
+  advanced: true
+- display: Retries
+  name: retries
+  type: 0
+  defaultvalue: 5
   required: false
```

Comment:
Since we're changing the ID of this integration (which will force customers to configure a new instance anyway), I removed the `access_key` and `secret_key` fields which are replaced by the type-9 `credentials` field and are kept for BC.